### DISCUSSION TITLE
Good day, How can i continuously connect to an API to get live data upon changes or some data are added?

### DISCUSSION THREAD

**Discussion by noelanthonydiaz** [TanStack/query#4817] [Q&A]
(ex. youtube live stream comments or facebook) 
 i was planning to use this if possible
 
[https://developers.facebook.com/docs/graph-api/server-sent-events/endpoints/live-comments/](https://developers.facebook.com/docs/graph-api/server-sent-events/endpoints/live-comments/)

**Comment by TkDodo** [comment]
Have you read: https://tkdodo.eu/blog/using-web-sockets-with-react-query
